itou/approvals/models.py

```diff
@@ -125,6 +125,9 @@ class Approval(CommonApprovalMixin):
     # This prefix is used by the ASP system to identify itou as the issuer of a number.
     ASP_ITOU_PREFIX = settings.ASP_ITOU_PREFIX
 
+    # The period of time during which it is possible to prolong a PASS IAE before it ends.
```

Comment:
J'avais encore jamais vu / entendu `prolong` !
Extend ?

Reply:
J'ai hésité mais j'ai choisi ***to prolong*** pour éviter toute ambiguïté avec le nom du modèle et le système de PE [qui permet _prolongation_ et _extension_](https://github.com/betagouv/itou/blob/76b26f61711c8f3b6c01710148862fa6d192bb6d/itou/approvals/models.py#L600-L608).

![prolonger](https://user-images.githubusercontent.com/281139/108731311-b88a1500-752c-11eb-82ab-557619847f36.png)